test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/AbstractService.java

```diff
@@ -123,6 +123,7 @@ public String getID() {
         final Map<String, Object> bookkeeperSpec = ImmutableMap.<String, Object>builder().put("image",
                                                                                              getImageSpec(DOCKER_REGISTRY + PREFIX + "/" + BOOKKEEPER_IMAGE_NAME, PRAVEGA_BOOKKEEPER_VERSION))
                                                                                         .put("replicas", bookieCount)
+                                                                                        .put("resources", getResources("2000m", "5Gi", "1000m", "3Gi"))
```

Comment:
Please, could you set the memory `request` to `2Gi` and the memory `limit` to 4Gi`? This is how it is set by QE team and will lower the probabilities to tun out of resources in a cluster running system tests.